apps/passport-client/src/util.ts

```diff
@@ -74,7 +74,7 @@ function getVerifyUrlPrefixes(): string[] {
 
 // Given an input string, check if there exists a ticket verify URL within it.
 // If so, return the last occurance of a verify URL. If not, return null.
-export function getLastValidVerifyUrl(inputString: string): string {
+export function getLastValidVerifyUrl(inputString: string): string | null {
```

Comment:
```suggestion
export function getLastValidVerifyUrl(inputString: string): string | unefined {
```

i generally avoid introducing `null` into this codebase and use `undefined` for consistency everywhere

Reply:
The `null` was already there in the final return statement, this change just makes the function signature accurate. I can probably change the `null` to `undefined` though.